server/modules/analysis/service/rChainService/rFile/rFile.js

```diff
@@ -3,7 +3,7 @@ import * as FileUtils from '@server/utils/file/fileUtils'
 
 import { arenaInfo } from '../rFunctions'
 
-const _lineSeparator = `${StringUtils.NEW_LINE}${StringUtils.NEW_LINE}`
+const _doubleFeed = `${StringUtils.NEW_LINE}${StringUtils.NEW_LINE}`
```

Comment:
What about contentSeparator? It tells what it does...

Reply:
double feed no?

Reply:
ok

Reply:
ok done

Reply:
ok done